### ISSUE TITLE
Moving Clouds: DeserializationError: Deserialization error: No JSON object could be decoded

### ISSUE THREAD

**Issue by sako0938** [JrCs/docker-letsencrypt-nginx-proxy-companion#573]
Hi, I am trying to move clouds from DigitalOcean to AWS, and I am getting some issues with DeserializationError. I am only allowing ports 22, 443, and 80. I have tried a few different older version of nginx-proxy and docker-letsencrypt-nginx-proxy-companion and cant get anything to work. Any ideas for the issue or potential ways to continue debugging?

docker logs ssl-helper
```
Info: Custom Diffie-Hellman group found, generation skipped.
Reloading nginx proxy (c4c0119e1432b05b308982729ff558d016ef7ad797ecacf83484b4c314415243)...
2019/08/19 19:26:51 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification ''
2019/08/19 19:26:51 [notice] 40#40: signal process started
2019/08/19 19:26:52 Generated '/app/letsencrypt_service_data' from 3 containers
2019/08/19 19:26:52 Running '/app/signal_le_service'
2019/08/19 19:26:52 Watching docker events
2019/08/19 19:26:52 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
/etc/nginx/certs/api.dev-babylon.com /app
Reloading nginx proxy (c4c0119e1432b05b308982729ff558d016ef7ad797ecacf83484b4c314415243)...
2019/08/19 19:26:52 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification ''
2019/08/19 19:26:52 [notice] 62#62: signal process started
Creating/renewal api.test-domain.com certificates... (api.test-domain.com)
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1584, in main
    return main_with_exceptions(cli_args)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1561, in main_with_exceptions
    existing_data = load_existing_data(args.ioplugins)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1322, in load_existing_data
    all_data = IOPlugin.registered[plugin_name].load()
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 346, in load
    return self.load_from_content(content)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 465, in load_from_content
    account_key=self.load_jwk(content),
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 375, in load_jwk
    return jose.JWKRSA.json_loads(data)
  File "/usr/lib/python2.7/site-packages/josepy/interfaces.py", line 178, in json_loads
    raise errors.DeserializationError(error)
DeserializationError: Deserialization error: No JSON object could be decoded

Unhandled error has happened, traceback is above

Debugging tips: -v improves output verbosity. Help is available under --help.
/app
Sleep for 3600s
2019/08/19 19:27:01 Received event die for container c3702ce6dea0
2019/08/19 19:27:01 Received event stop for container c3702ce6dea0
2019/08/19 19:27:03 Received event start for container 2de698187784
2019/08/19 19:27:18 Debounce minTimer fired
2019/08/19 19:27:18 Generated '/app/letsencrypt_service_data' from 3 containers
2019/08/19 19:27:18 Running '/app/signal_le_service'
/etc/nginx/certs/api.test-domain.com /app
Creating/renewal api.test-domain.com certificates... (api.test-domain.com)
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1584, in main
    return main_with_exceptions(cli_args)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1561, in main_with_exceptions
    existing_data = load_existing_data(args.ioplugins)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 1322, in load_existing_data
    all_data = IOPlugin.registered[plugin_name].load()
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 346, in load
    return self.load_from_content(content)
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 465, in load_from_content
    account_key=self.load_jwk(content),
  File "/usr/lib/python2.7/site-packages/simp_le.py", line 375, in load_jwk
    return jose.JWKRSA.json_loads(data)
  File "/usr/lib/python2.7/site-packages/josepy/interfaces.py", line 178, in json_loads
    raise errors.DeserializationError(error)
DeserializationError: Deserialization error: No JSON object could be decoded

Unhandled error has happened, traceback is above

Debugging tips: -v improves output verbosity. Help is available under --help.
/app
Sleep for 3600s
```

**Comment by buchdag** [comment]
Did you import old data from DigitalOcean ? The ACME account key seems to have an issue.

**Comment by sako0938** [comment]
I was able to delete 
certs
vhost.d
challenge
folders which were mounted from my filesystem. That got it going correctly, though I didn't copy those folders from my DO deployment. In my debugging I reverted to: jrcs/letsencrypt-nginx-proxy-companion:v1.11.2, so I was still using that image when I deleted and retried.
I can close. I might get a chance to work on it later, and I can try deleting again and running with latest.
